sonar-db/src/main/java/org/sonar/db/user/UserPermissionDto.java

```diff
@@ -22,49 +22,36 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
-/**
- * @since 3.2
- */
-public class UserRoleDto {
-  private Long id;
+public class UserPermissionDto {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)